httpx/httpx_test.go

```diff
@@ -47,15 +47,15 @@ func TestIntegrationSetResolver(t *testing.T) {
 	}
 }
 func TestSetCABundle(t *testing.T) {
-	client := httpx.NewClient(handlers.NoHandler)
+	client := httpx.NewClientWithoutProxy(handlers.NoHandler)
 	err := client.SetCABundle("../testdata/cacert.pem")
 	if err != nil {
 		t.Fatal(err)
 	}
 }
 
 func TestForceSpecificSNI(t *testing.T) {
-	client := httpx.NewClient(handlers.NoHandler)
+	client := httpx.NewClientWithoutProxy(handlers.NoHandler)
```

Comment:
Using the client without proxy for correctness. We're not using the client here, but, if we were, we'd be surprised by the proxy used to test the proxy functionality, so better to be future-proof.